src/pages/UserList/UserList.js

```diff
@@ -10,12 +10,42 @@ import {
 import React, { useState, useEffect } from 'react'
 import { useDebounce } from 'use-debounce'
 import PageHeader from '../../components/PageHeader.js'
+import { useFeatureToggle } from '../../hooks/useFeatureToggle.js'
 import navigateTo from '../../utils/navigateTo.js'
 import Filters from './Filters.js'
 import { useFilters } from './useFilters.js'
 import styles from './UserList.module.css'
 import UserTable from './UserTable.js'
 
+const getUsersQueryFilters = ({
+    organisationUnits,
+    emailVerificationStatus,
+    displayEmailVerifiedStatus,
+}) => {
+    const organisationUnitsFilter =
+        organisationUnits.length > 0
+            ? `organisationUnits.id:in:[${organisationUnits.map(
+                  ({ id }) => id
+              )}]`
+            : undefined
+    // email verification filter is ignored if feature is not accessible
```

Comment:
we allow people to copy/paste links with emailVerification query parameter, but it is ignored/not applied to the filter if the conditions for toggling this feature do not exist (this is similar to what would happen if you entered your own random query parameter in the URL, e.g. it would be ignored)